test/unit/util/fsTest.ts

```diff
@@ -226,4 +226,71 @@ describe('util/fs', () => {
       expect(exists).to.be.false;
     });
   });
+
+  describe('areFilesEqual', () => {
+    afterEach(() => {
+      $$.SANDBOX.restore();
+    });
+
+    it('should return true if the files are the same', async () => {
```

Comment:
I don't love writing files during tests... can we just stub out the fs calls to return data?

Reply:
Wanted to put our chat here just in case someone else is curious. @williamruemmele-sf 

> why do you prefer stubbing files instead of writing them?

>Because, IMO, unit test should be testing as small amount of code as possible. Instead of just testing your method, you are also testing that the fs  library reads/writes the files from the disk. That also makes test take longer. Really you are assuming that fs is going to return you something, and your method is going to react to that. That is what you should be testing.